raiden/tests/integration/long_running/flaky/test_token_networks.py

```diff
@@ -134,11 +113,16 @@ def test_participant_selection(raiden_network, token_addresses, skip_if_tester):
         ) for app in raiden_network
     ]
 
-    assert wait_for_saturated(
-        connection_managers,
-        registry_address,
-        token_address,
-    ) is True
+    unsaturated_connection_managers = connection_managers[:]
+    with gevent.Timeout(
```

Comment:
This changes the timeout from the previous block number based one to a time based one. How does this affect test reliability?

Reply:
The tests shouldn't be affected by it, as we expect the transactions to go sooner than later. The main change with that is that now, the error isn't hidden below 3 layers of utils that does the same thing, and gets prettyprinted, so we can know what went wrong more easily.

Reply:
Ok lets see what happens, if this test starts to act up we may need to revisit it.